cachito/workers/config.py

```diff
@@ -131,6 +150,26 @@ def validate_celery_config(conf, **kwargs):
         raise ConfigError('The configuration "cachito_api_url" must be set')
 
 
+def validate_nexus_config():
```

Comment:
As discussed, there's no clean way to determine what queues the Celery worker is listening on since the configuration can be overridden with the `-Q` option.